update-and-commit-translations.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Script to pull transifex translation files, and commit them to git if there are any changes.
+
+tx pull -f --mode=reviewed
+
+if [[ -z $(git diff-index --quiet HEAD --) ]]; then
```

Comment:
Sorry, I meant something like 

```
if ! git diff-index --quiet HEAD --; then 
  echo 'There are changes to be committed'
  #git commit and push
fi
```

`git diff-index --quiet HEAD --` will never print anything to the output, hence `-z $(git diff-index --quiet HEAD --)` should always be true. 

On this case, it's better to test the exit code than the output.